src/lightning/pytorch/loggers/comet.py

```diff
@@ -302,12 +303,14 @@ def experiment(self) -> Union["Experiment", "ExistingExperiment", "OfflineExperi
 
         return self._experiment
 
+    @override
```

Comment:
why not consistenly put it below?

Reply:
See https://github.com/Lightning-AI/lightning/pull/18934#discussion_r1382188208. PEP-698 states that `@override` should go below descriptor-based decorators like `@property`, but above wrapper-based decorators. `rank_zero_only()` seems to be a wrapper function, hence why I put `@override` above it.